features/users/users.feature

```diff
@@ -150,27 +150,9 @@ Feature: Welcome Page
     Then I should see Personal Details page
     When I click on Update
     Then I should see the user edit page
-    When I fill the user_dob field with value '9999999'
+    When I fill the user user date of birth field with future date
     And I submit the user details
-    Then I should see 'Invalid format' error on edit page
-    When I fill the user_dob field with value '34-43-8349'
-    And I submit the user details
-    Then I should see 'Invalid format' error on edit page
-
-# TODO(DoxasticFox): Re-enable these as part of Australian-Genomics/CTRL#5
-#  Scenario: User should see errors for dob in future
-#    Given I exist as a user
-#    And I am not logged in
-#    When I sign in with valid credentials
-#    Then I should be signed in
-#    And I should see the dashboard page
-#    When I click on My Personal Details
-#    Then I should see Personal Details page
-#    When I click on Update
-#    Then I should see the user edit page
-#    When I fill the user user date of birth field with future date
-#    And I submit the user details
-#    Then I should see date from the future error on edit page
+    Then I should see date from the future error on edit page
```

Comment:
It's a bit unclear from the diff, but I deleted this test:

```
Scenario: User should see errors for invalid date format for dob
```

...and I enabled/uncommented the previously uncommented one:

```
Scenario: User should see errors for dob in future
```

The test I deleted was for a state which the new datepicker makes impossible.